pkg/services/provisioning/dashboards/dashboard.go

```diff
@@ -24,7 +23,7 @@ type DashboardProvisioner interface {
 }
 
 // DashboardProvisionerFactory creates DashboardProvisioners based on input
-type DashboardProvisionerFactory func(string, dashboards.Store, plugins.DataRequestHandler) (DashboardProvisioner, error)
+type DashboardProvisionerFactory func(string, dashboards.Store) (DashboardProvisioner, error)
```

Comment:
plugins.DataRequestHandler did not seem used anywhere for dashboard provisioning.
However I might have missed something. Can someone confirm this is a useless parameter?